src/py42/_internal/clients/storage/storagenode.py

```diff
@@ -0,0 +1,29 @@
+from py42.clients import BaseClient
+import requests
+
+
+class StorageNodeClient(BaseClient):
+
+    _base_uri = u"c42api/v3/"
+
+    def __init__(self, session, archive_guid, file_id, timestamp):
+        super(StorageNodeClient, self).__init__(session)
+        self._archive_guid = archive_guid
+        self._file_id = file_id
+        self._timestamp = timestamp
+
+    def get_download_token(self):
+
+        params = "archiveGuid={0}&fileId={1}&versionTimestamp={2}".format(
+            self._archive_guid, self._file_id, self._timestamp
+        )
+        resource = u"FileDownloadToken"
+        uri = "{0}{1}?{2}".format(self._base_uri, resource, params)
+        return self._session.get(uri)
+
+    def get_file(self, host_address, token, file):
```

Comment:
Should we write content to a file or stdout??

Reply:
No, we should just return the stream. The user can decide where to save it on their own.

There is an example in the README we can append to explaining how to do this

Reply:
Yeah, the user may want to do something else with the stream besides writing to a file, (e.g. making it streamable through a browser or printing the raw stream to a terminal), so returning the stream is the expected implementation. This is what we do for `stream_file_from_backup`.

Reply:
Changed.